addons/purchase_stock/models/purchase.py

```diff
@@ -406,12 +406,13 @@ def _update_move_date_deadline(self, new_date):
     def _create_or_update_picking(self):
         for line in self:
             if line.product_id and line.product_id.type in ('product', 'consu'):
+                prec = line.product_uom.rounding
```

Comment:
Fix is ok, but this name is a bit confusing since prec can be interpreted for both `precision_digits` and `precision_rounding`. I would say either name it `rounding` or don't create a variable at all since it's only for 2 uses anyways

Reply:
Changed it to `rounding`. The other approach makes the line ridiculously long, and splitting `if`s over multiple lines is never not awkward

Reply:
:thinking: it doesn't need to be split. Your linter might be set too strictly for the line length